src/DataDefinitionsBundle/Model/ExportDefinition.php

```diff
@@ -21,6 +21,11 @@
  */
 class ExportDefinition extends AbstractDataDefinition implements ExportDefinitionInterface
 {
+    /**
+     * @var bool
+     */
+    public $enableInheritance = true;
```

Comment:
[note] Pimcore enables inheritance in most cases - for CLI and non-admin contexts.